2 - ListsLogicAndIterations/Code/Dictionarys.cs

```diff
@@ -7,16 +7,31 @@ public static class Dictionarys
 	{
 		public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
 		{
+            var Dict = new Dictionary<Guid, decimal>();
+            Dict.Add(key, value);
+            return Dict;
```

Comment:
Though this does work, you can short hand all of this by

`return new Dictionary<Guid, decimal> { {key, value} };`